### PR TITLE
Cherry-pick some flavor fixes

### PR DESCRIPTION
**Pull request by quardbreak**

### CODE REVIEW

**Review by PsiOmegaDelta — DISMISSED** [review]

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 	layer = SIDE_WINDOW_LAYER
 	anchored = 1.0
-	atom_flags = ATOM_FLAG_NO_TEMP_CHANGE | ATOM_FLAG_CHECKS_BORDER
```
Comment (MistakeNot4892) [inline code/game/objects/structures/window.dm:10]: This flag only exists for windows as far as I can tell - what is removing it meant to fix?

File: code/game/turfs/turf.dm [hunk 2]
```diff
@@ -269,7 +269,7 @@ var/const/enterloopsanity = 100
 
 /turf/proc/remove_cleanables()
 	for(var/obj/effect/O in src)
-		if(istype(O,/obj/effect/rune) || istype(O,/obj/effect/decal/cleanable) || istype(O,/obj/effect/overlay))
```
Comment (MistakeNot4892) [inline code/game/turfs/turf.dm:272]: Why remove this?

File: code/modules/mob/mob.dm [hunk 3]
```diff
@@ -461,7 +461,8 @@
 // If usr != src, or if usr == src but the Topic call was not resolved, this is called next.
 /mob/OnTopic(mob/user, href_list, datum/topic_state/state)
 	if(href_list["flavor_more"])
-		show_browser(user, "<HTML><HEAD><TITLE>[name]</TITLE></HEAD><BODY><TT>[replacetext(flavor_text, "\n", "<BR>")]</TT></BODY></HTML>", "window=[name];size=500x200")
+		var/text = "<HTML><HEAD><TITLE>[name]</TITLE></HEAD><BODY><TT>[replacetext(flavor_text, "\n", "<BR>")]</TT></BODY></HTML>"
```
Comment (MistakeNot4892) [inline code/modules/mob/mob.dm:464]: All this does is add an alloc to the logic, why change it?

File: code/game/turfs/turf.dm (hunk 2 above)
Comment (MistakeNot4892) [reply code/game/turfs/turf.dm:272]: Nvm.

File: code/modules/mob/mob.dm (hunk 3 above)
Comment (afterthought2) [reply code/modules/mob/mob.dm:464]: There's a dm-level issue with macro substitutions and string logic. Go figure.

See the bay PR for an extensive discussion.

File: code/game/objects/structures/window.dm (hunk 1 above)
Comment (afterthought2) [reply code/game/objects/structures/window.dm:10]: Seems to be about forbidding movement onto low walls. Probably the correct fix is to set this if `dir` is a cardinal, and unset it for full-tile windows (don't recall if they are `dir = 0` or `dir = 5`).

Comment (MistakeNot4892) [reply code/game/objects/structures/window.dm:10]: Tested this locally and it doesn't seem to impact window behavior at all.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**